_docs/results/sources.md

```diff
@@ -4,36 +4,10 @@ category: Results
 
-<p>
-    To do that, DuckDuckGo gets its results from over
-    <a href="https://duck.co/ia">four hundred sources</a>. These include hundreds
-    of vertical sources delivering niche Instant Answers, <a href="https://duckduckgo.com/duckduckbot">DuckDuckBot</a> (our
-    crawler) and crowd-sourced sites (like Wikipedia, stored in our answer
-    indexes). We also of course have more traditional links in the search results, which we also source from multiple partners, though most commonly from Bing (and none from Google).
-</p>
+To do that, DuckDuckGo gets its results from over four hundred sources. These include hundreds of vertical sources delivering niche Instant Answers, [DuckDuckBot](https://duckduckgo.com/duckduckbot) (our crawler) and crowd-sourced sites (like Wikipedia, stored in our answer indexes). We also of course have more traditional links in the search results, which we also source from multiple partners, though most commonly from Bing (and none from Google).
 
-<p>
-    We continue to partner with more and more sources to bring you the best
-    Instant Answers from the best sources. Our focus is on synthesizing it all
-    together into a superior search experience.
-</p>
+We continue to partner with more and more sources to bring you the best Instant Answers from the best sources. Our focus is on synthesizing it all together into a superior search experience.
 
-Partners and Privacy: As per our strict
-<a href="https://duckduckgo.com/privacy">privacy policy</a>, we never share any
-personal information with any of our partners. The way it works is when we call
-a partner for information, it is proxied through our servers so it stays
-completely anonymous. That is, any call to a partner looks to the partner as it
-is from us and not the user itself, and no user personal information is passed
-in that process (e.g. their IP address). That way we can build our search result
-pages using these 100s of partner sources, while still keeping them completely
-anonymous to you.
+Partners and Privacy: As per our strict [privacy policy](https://duckduckgo.com/privacy), we never share any personal information with any of our partners that could lead to the creation of search histories. The way it works is when we call a partner for information used in search results, it is proxied through our servers so it stays anonymous. That is, any call to a partner looks to the partner as it is from us and not the user itself, and no unique identifier is passed in that process (e.g., their IP address). That way we can build our search result pages using these 100s of partner sources, while still keeping them anonymous to you.
```

Comment:
Updated copy here

Reply:
Could possibly remove "The way it works is" here, to make the copy shorter while keep the meaning.

Also, seems to be missing a "though":
> looks to the partner as though it is from us